### ISSUE TITLE
[patch] explicitly include linux.h in mod_tile to compile for gcc 4.3

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#914]
**[Submitted to the original trac issue database at 1.46am, Tuesday, 20th May 2008]**

This patch explicitly adds #include <linux/linux.h> to speedtest.cpp and render_list.c in order to get mod_tile to compile with gcc 4.3.0.

**Comment by tomhughes** [comment]
**Author: tom@compton.nu**
**[Added to the original trac issue at 7.09am, Tuesday, 20th May 2008]**

Can you clarify what it is in the code that requires the inclusion of linux/linux.h exactly - it shouldn't normally be necessary to include that in any user code, so I would like to see if there is a more portable solution to the problem you are encountering.

**Comment by tomhughes** [comment]
**Author: tom@compton.nu**
**[Added to the original trac issue at 7.47am, Tuesday, 20th May 2008]**

OK - I think I've sorted this. I assume it was the PATH_MAX definition you were having problems with? That's what I was seeing on Fedora 9 though I'm not sure if gcc 4.3 or glibc 2.8 was the trigger.

The correct fix is to include limits.h rather than a linux specific file, and I have now committed that as r7866.
